dora/core/common/src/main/java/alluxio/wire/WorkerNetAddress.java

```diff
@@ -56,6 +56,9 @@ public final class WorkerNetAddress implements Serializable {
   @Expose
   @com.google.gson.annotations.SerializedName("DomainSocketPath")
   private String mDomainSocketPath = "";
+  @Expose
+  @com.google.gson.annotations.SerializedName("HttpServerPort")
+  private int mHttpServerPort;
```

Comment:
This port is for the restful API server on the worker, right? There is already a "web port" which runs the web UI. Since both the UI and the restful API runs over HTTP, and possibly more services that run over HTTP will be added, can we be more specific that this port is used for the restful API exclusively?

```suggestion
  @com.google.gson.annotations.SerializedName("RestApiPort")
  private int mRestApiPort;
```

Reply:
The HttpServerPort is used in the `PropertyKey` as the config for the restful API. The name is `WORKER_HTTP_SERVER_PORT`.  If we change the `mHttpServerPort` to `mRestApiPort`, it will cause naming inconsistencies since other WorkerNetAddress variables (e.g. mWebPort, mDataPort) follow the same names as configs in the `PropertyKey`.

Reply:
I suggest keeping the name for now. If we want to change the name in the future, we'll need to change both this one and the property key (and probably any other places using HTTP server port...).

Reply:
can u also add some comment here, saying this is a "optional"-like field which will not be included in equalness comparison